### PR TITLE
Type script

### PR DESCRIPTION
**Pull request by 0xsarvesh**
Typescript implementation of dost app.

### CODE REVIEW

**Comment by puneet-khushwani-eth** [comment]
Started reviewing.

File: package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "styled-components": "4.3.2",
     "typescript": "^3.5.3",
     "web3": "2.0.0-alpha",
-    "web3-utils": "2.0.0-alpha"
+    "web3-utils": "2.0.0-alpha",
+    "web3-eth-accounts": "1.2.1"
```
Comment (deepesh-kn) [inline package.json]: Why do we need to import this? 
`web3.eth.account` can be used form web3?

File: src/KeyManager/Account.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import Web3 from "web3";
+import localStorage from "../services/LocalStorage";
+
+export class Account {
+
+  static new() {
+    return (new Web3('')).eth.accounts.create();
+  }
+
+  static fromPrivateKey(privateKey) {
```
Comment (deepesh-kn) [inline src/KeyManager/Account.ts]: Please add return types (check other places in this file)

**Review by deepesh-kn — COMMENTED** [review]
Overall LGTM,
The return types are not defined for many functions.

**Review by deepesh-kn — APPROVED** [review]
LGTM 👍

**Pull request merged**